src/components/daily/smbgtooltip/SMBGTooltip.js

```diff
@@ -0,0 +1,149 @@
+/*
+ * == BSD2 LICENSE ==
+ * Copyright (c) 2016, Tidepool Project
+ *
+ * This program is free software; you can redistribute it and/or modify it under
+ * the terms of the associated License, which is identical to the BSD 2-Clause
+ * License as published by the Open Source Initiative at opensource.org.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the License for more details.
+ *
+ * You should have received a copy of the License along with this program; if
+ * not, you can obtain one from Tidepool Project at tidepool.org.
+ * == BSD2 LICENSE ==
+ */
+
+import React, { PropTypes, PureComponent } from 'react';
+import _ from 'lodash';
+import { classifyBgValue, reshapeBgClassesToBgBounds } from '../../../utils/bloodglucose';
+import { formatLocalizedFromUTC } from '../../../utils/datetime';
+import Tooltip from '../../common/tooltips/Tooltip';
+import colors from '../../../styles/colors.css';
+import styles from './SMBGTooltip.css';
+
+const medtronic600BGMessages = {
```

Comment:
Is there a clean way for us to abstract these variables to `bgMessages` and `calibrationMessages` to avoid having device-specific details in the component?  

Perhaps moving `getMedtronic600AnnotationMessages` to it's own utility (maybe taking in the manufacturer key to be device-agnostic?) and passing them in via a `messages` prop. 

Thoughts?

Reply:
Makes sense.